### PR TITLE
Only distinguish live and non-live when output xrefmap

### PR DESCRIPTION
**Pull request by 928PJY**
#6448 

[AB#291373](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/291373)
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6450)

### CODE REVIEW

**Review by herohua — APPROVED** [review]

File: docs/specs/xref.yml [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ repos:
 outputs:
   docs/a.json:
   .xrefmap.json: |
-    {"references":[{"uid":"a", href: "https://docs.com/docs/a?branch=master"}]}
+    {"references":[{"uid":"a", href: "https://docs.com/docs/a?branch=test"}],"properties": {"tags": ["/","internal"]}}
```
Comment (live1206) [inline docs/specs/xref.yml:1240]: property tags can be omitted from the test

**Review by live1206 — APPROVED** [review]

File: src/docfx/build/xref/XrefResolver.cs [hunk 2]
```diff
@@ -145,7 +145,7 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                 {
```
Comment (yufeih) [inline src/docfx/build/xref/XrefResolver.cs:145]: Apply this patch only when `outputUrlType == Docs`?

**Review by yufeih — APPROVED** [review]

File: src/docfx/build/xref/XrefResolver.cs (hunk 2 above)
Comment (928PJY) [reply src/docfx/build/xref/XrefResolver.cs:145]: Why should add this logic in this time, I mean the original logic is also applied to all the `outputUrlType`. and in our spec, we already consume the live branch

---
In reply to: [477133335](https://github.com/dotnet/docfx/pull/6450#discussion_r477133335) [](ancestors = 477133335)

Comment (yufeih) [reply src/docfx/build/xref/XrefResolver.cs:145]: doesn't need to be in this PR

**Pull request merged**